.github/CONTRIBUTING.md

```diff
@@ -57,21 +57,21 @@ actual development.
  * While we have no issue helping contributors (and especially new contributors) bring reasonably sized contributions up to standards via the pull request review process, larger contributions are expected to pass a higher bar of completeness and code quality *before* you open a pull request. Maintainers may close such pull requests that are deemed to be substantially flawed. You should take some time to discuss with maintainers or other contributors on how to improve the changes.
 
 #### Using Changelog
- * Tags used in changelog include add/rscadd, del/rscdel, fix/fixes, typo/spellcheck.
+ * The tags able to be used in the changelog are: `add/soundadd/imageadd`, `del/sounddel/imagedel`, `tweak`, `fix`, `wip`, `spellcheck`, and `experiment`.
  * Without specifying a name it will default to using your GitHub name.
  Some examples
```

Comment:
While you're here:
```suggestion
 Some examples include:
```
Also if you can, maybe add some space between this text and the examples below. They look a bit smushed together: 

![image](https://user-images.githubusercontent.com/42044220/126927832-b65ea2c5-6842-441d-ba7f-a08d1a7fcfc6.png)

Reply:
What do you mean?
![image](https://user-images.githubusercontent.com/57483089/126963899-a2b08649-5c6f-4ea4-acb1-19f37ccfdcf0.png)

Reply:
I just meant the space between the "Without specifying" line and the code block below was very small. Pretty minor issue though.

Reply:
Oh, yeah I see what you mean now. The spacing is smaller if it's under a bullet list than a regular paragraph. I'll add an extra line on those. 👍